ProgrammaticUI/Controllers/HomeViewController.swift

```diff
@@ -2,12 +2,13 @@ import UIKit
 
 class HomeViewController: UIViewController {
 
-    let stackView = UIStackView()
-    private var kmsLabel: UILabel!
-    private var caloriesLabel: UILabel!
-    private var durationLabel: UILabel!
-    let imageFitness = SAImageView(imageNamed: "fitness.png")
-    var insertData = InsertDataViewController()
+    private let stackView = UIStackView()
+    private let summaryLabel = SALabel(texto: "Summary", font: .boldSystemFont(ofSize: 40))
+    private var kmsLabel = SALabel(texto: "Kms: 0.0", font: .systemFont(ofSize: 20))
+    private var caloriesLabel = SALabel(texto: "Calories: 0", font: .systemFont(ofSize: 20))
+    private var durationLabel = SALabel(texto: "Duration: 00:00:00", font: .systemFont(ofSize: 20))
+    private let imageFitness = SAImageView(imageNamed: "fitness.png")
```

Comment:
I would create a static property in `SAImageView` to store all these `imageNamed`. This way, if a specific image is renamed it would only need a change in one place.
```suggestion
    private let imageFitness = SAImageView.fitnessImage
```